### DISCUSSION TITLE
Connect to https://launchermeta.mojang.com:443 failed: Connection timed out

### DISCUSSION THREAD

**Discussion by mrcaidev** [itzg/docker-minecraft-server#1343] [Q&A] [answered]
I started the container with default options by `docker run -d -it -p 25565:25565 -e EULA=TRUE itzg/minecraft-server`, and was constantly checking the health status with `docker container inspect -f "{{.State.Health.Status}}"`. After a period of `starting`, it shows `unhealthy`; and when I type `docker logs`, it shows logs like:

```txt
[init] Running as uid=1000 gid=1000 with /data as 'drwxrwxr-x 2 1000 1000 4096 Feb  3 04:28 /data'
[mc-image-helper] 17:31:39.015 ERROR : Operation failed: Connect to https://launchermeta.mojang.com:443 [launchermeta.mojang.com/13.226.223.20] failed: Connection timed out
[init] ERROR: version lookup failed: 
```

I tried 3 times and it all shows connection timeout. Seems that this is a network problem, but everything goes well when I visit Internet...

Any idea about this problem? Or is there any other ways to download server resources myself and skip downloading when starting container?

**Comment by itzg** [comment, marked as answer]
It sounds like a networking problem in your docker installation and/or firewall. If the container can't connect to Mojang then several steps after that are going to fail too. 

If your host has no network connectivity then you can use TYPE=CUSTOM and provide a pre downloaded server jar...but if networking is not working then defeats the purpose of a server.

**Comment by JamesDimonaco** [comment]
Hi, I'm getting a similar error that i've been banging my head over for a while 

i'm not sure if it's network related and if it's my end or where i'm trying to pull the image 

`[init] Running as uid=1000 gid=1000 with /data as 'drwxrwxrwx 2 1000 1000 4096 May 17 19:30 /data'
[mc-image-helper] 20:05:05.106 ERROR : 'get' command failed: launchermeta.mojang.com: Temporary failure in name resolution
[init] ERROR: version lookup failed:`

**Reply by itzg** [reply]
Yours is only similar in that it is encountering an error on the same, initial external network operation. 

In your case

```
Temporary failure in name resolution
```

indicates that DNS is misconfigured on your system. I would recommend searching around for docker/container DNS configuration ideas.
